server/service/src/sync/translations/vvm_status.rs

```diff
@@ -0,0 +1,95 @@
+use repository::{vvm_status_row::{VVMStatusRow, VVMStatusRowDelete}, StorageConnection, SyncBufferRow};
+
+use serde::Deserialize;
+
+use super::{PullTranslateResult, SyncTranslation};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize)]
+pub struct LegacyVVMStatusRow {
+    ID: String,
+    description: String,
+    code: String,
+    level: i32,
+    is_active: bool,
+    unusable: bool,
+    option_id: String,
```

Comment:
If we make option optional, it think that will be more correct with older mSupply versions.
At the moment the option_id will be set to "" which isn't correct, it should be `null` in the database
```suggestion
    #[serde(deserialize_with = "empty_str_as_option_string")]
    option_id: Option<String>,
```